tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -238,6 +238,10 @@ public static string Defaultx5cValue
         public const string MsalArlingtonCCAKeyVaultUri = "https://msidlabs.vault.azure.net:443/secrets/ARLMSIDLAB1-IDLASBS-App-CC-Secret";
         public const string MsalArlingtonCCAKeyVaultSecretName = "ARLMSIDLAB1-IDLASBS-App-CC-Secret";
 
+        public const string ManagedIdentityClientId = "client_id";
+        public const string ManagedIdentityObjectId = "object_id";
+        public const string ManagedIdentityResourceId = "mi_res_id";

```

Comment:
I'd use the ones defined in the `Constants` class.

Reply:
let me take a look at it.